Content.Shared/Chemistry/ReagentPrototype.cs

```diff
@@ -43,13 +47,36 @@ public void LoadFrom(YamlMappingNode mapping)
             serializer.DataField(ref _id, "id", string.Empty);
             serializer.DataField(ref _name, "name", string.Empty);
             serializer.DataField(ref _description, "desc", string.Empty);
+            serializer.DataField(ref _visualDescription, "visualDesc", string.Empty);
```

Comment:
following on from the above (about colour localisation), I'd prefer this renamed to `ColorDescription`, so that people don't repeat the mistake of including 'a/an' in the description.

Both in C# and yaml pls.